__tests__/components/ui/Checkbox.test.tsx

```diff
@@ -0,0 +1,46 @@
+import Checkbox from "@/components/ui/Checkbox";
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("Checkbox", () => {
+    it("should render the checkbox", () => {
+        render(
+        <Checkbox/>
+        )
+        const checkbox = screen.getByTestId("checboxId"); 
+        expect(checkbox).toBeInTheDocument()
+    })
+
+    it("should render the checkbox with community style", () => {
+        render(<Checkbox communityStyles/>)
+        const checkboxStyle = screen.getByTestId("checboxId"); 
+        expect(checkboxStyle).toBeInTheDocument()
+        expect(checkboxStyle).toHaveStyle({color: "#0000FF"})
```

Comment:
We need to avoid testing the exact community color, we can only test of the color is not undefined or empty string. This is because the community color can changes at anytime which in this case will break the test.